app/services/apply_district.rb

```diff
@@ -100,10 +100,6 @@ def create_ecs_cluster
     aws.ecs.create_cluster(cluster_name: district.name)
   end
 
-  def create_or_update_network_stack
```

Comment:
not related to change set but I found that this method is not used so I just deleted it